lib/dns/src/index.ts

```diff
@@ -121,11 +121,11 @@ export function* ensureDNSExists({
 
   const clusters: string[] = yield call([
     opstraceClient,
-    opstraceClient.GetAll
+    opstraceClient.getAllEntries
```

Comment:
return type of `getAllEntries` definition:
```
getAllEntries(): Promise<unknown>
```

Reply:
Great example for how the redux-saga approach here with `yield call()` here is rather transparent for tsc -- no compile error thrown, despite incompatible types.

Reply:
Related: https://github.com/opstrace/opstrace-prelaunch/issues/1445 (`(un)installer: do without redux-saga`)

Reply:
https://github.com/redux-saga/redux-saga/issues/1504
And then of course there is
https://github.com/agiledigital/typed-redux-saga

Reply:
quick proof of concept, can indeed get to a `Type 'unknown' is not assignable to type 'string[]'.ts(2322)` with using https://github.com/agiledigital/typed-redux-saga.